src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -525,6 +529,8 @@ describe('MethodologySummary', () => {
     });
   });
 
+  // TODO EES-2156 - add tests for deleting the first draft version of a
```

Comment:
You could create a task list on this PR if there's a few things still outstanding?